### PR TITLE
Database

### PR DESCRIPTION
**Pull request by psriraj17**

### CODE REVIEW

**Comment by psriraj17** [comment]
look out for .py files and please review the code

File: postgre_insertion_sentences.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    conn_string = "host='localhost' dbname='postgres' user='postgres' password='Post$123'"
+
+    con = psycopg2.connect(conn_string)
+    curs = con.cursor()
+
+
+    for sentence in sentences:
+        curs.execute(f"INSERT INTO english (word, sentence) values ('{word}','{sentence}') ")
+    con.commit()
+    x = "commited row successfully"
```
Comment (rebelDi) [inline postgre_insertion_sentences.py:14]: Good call on returning the message if row was successfully commited

File: preprocessing_of_corpus.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+    sent_tokenized = sentences(corpus_raw)
+
+    # Removing all the unnecessary things from the text
+    get_sentences_without_special_symbols = [remove(line) for line in sent_tokenized]
+
+    # Removing punctuations
+    get_sentences_without_punctuations = [pun(line.lower()) for line in get_sentences_without_special_symbols]
+
+    # Removing extras from the sentences
+    get_sentences_without_extras = [extras(sent) for sent in get_sentences_without_punctuations]
```
Comment (rebelDi) [inline preprocessing_of_corpus.py:132]: What is considered extras here?

**Review by rebelDi — APPROVED** [review]

File: preprocessing_of_corpus.py (hunk 2 above)
Comment (psriraj17) [reply preprocessing_of_corpus.py:132]: After replacing the unnecessary symbols with spaces there might be series of gaps/spaces. The extras can remove such spaces between.

**Pull request merged**